### ISSUE TITLE
office_mru does not work for newer osquery releases

### ISSUE THREAD

**Issue by d4rk-d4nph3** [osquery/osquery#7698]
# Bug report

### What operating system and version are you using?

Verified this issue on two Windows 10 builds.

1. Windows Version 10 Pro Build 19042
2. Windows 10 Pro Build 18363 

### What version of osquery are you using?

_office_mru_ query works fine on v5.2.3 but as soon as I installed v5.4.0, the scheduled query started giving zero results and upon further digging, v5.3.0 also has this issue.

### What steps did you take to reproduce the issue?

```
osquery> select * from office_mru;
```

### What did you expect to see?

+-------------+---------+------------------------------------------------------------+------------------+------------------------------------------------+
| application | version | path                                                       | last_opened_time | sid                                            |
+-------------+---------+------------------------------------------------------------+------------------+------------------------------------------------+
| Excel       | 16.0    | C:\Users\XXX\Downloads\ABC.xlsx                | 1658386292       | S-1-5-21-xxx-xxx-xxx-1104 |

### What did you see instead?

Empty result.

The code of [_office_mru_](https://github.com/osquery/osquery/blob/master/osquery/tables/applications/windows/office_mru.cpp) is untouched for a couple of years but some changes might have brought this regression.

**Comment by iko1** [comment]
@d4rk-d4nph3, have you tried to run the query on multiple endpoints?
Which office version is installed on the endpoint and the query returns an empty result?

**Comment by d4rk-d4nph3** [comment]
Yes on multiple OS and Office versions (Office 2010 and the latest v16 release). Downgrading to v5.2.3 solves the issue.

**Comment by Samuel-Roach** [comment]
I've also seen this on a Windows 11 build:

- Microsoft Windows 11 Enterprise Build 22000

The issue seems to have appeared between v5.2.3 and v5.3.0 as stated above. It currently persists to v5.6.0.

**Comment by d4rk-d4nph3** [comment]
Bug still exists in the latest v5.8.1. Tested on Windows 10 Pro 21H1 19043.2364.

**Comment by willjjohnson1** [comment]
Still seeing this in v5.11.0 on Windows 11 Pro 23H2 Build 22631.

I have not tested the latest v5.12.1 or another computer.

**Comment by wjohnson-soclogix** [comment]
Still persists in v5.13.1.
Tested on Windows 11 Pro 22H2 Build 22621.
